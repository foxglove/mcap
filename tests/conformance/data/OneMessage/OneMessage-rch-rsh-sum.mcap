version https://git-lfs.github.com/spec/v1
oid sha256:899415d1ad729747a88885460add7e050df3845b5916c81f6ea137631963e0fb
size 323
