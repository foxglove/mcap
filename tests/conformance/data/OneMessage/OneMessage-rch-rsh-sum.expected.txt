Header library="" profile=""
Schema id=1 schema=b schema_encoding=c schema_name=Example
ChannelInfo id=1 message_encoding=a metadata={foo=bar} schema_id=1 topic=example
Message channel_id=1 log_time=2 message_data=<010203> publish_time=1 sequence=10
Schema id=1 schema=b schema_encoding=c schema_name=Example
ChannelInfo id=1 message_encoding=a metadata={foo=bar} schema_id=1 topic=example
SummaryOffset group_length=32 group_opcode=3 group_start=151
SummaryOffset group_length=47 group_opcode=4 group_start=183
SummaryOffset group_length=0 group_opcode=13 group_start=230
SummaryOffset group_length=0 group_opcode=10 group_start=230
SummaryOffset group_length=0 group_opcode=8 group_start=230
Footer summary_crc=0 summary_offset_start=230 summary_start=151
